### PR TITLE
New seed helm chart, runtime genesis

### PR DESCRIPTION
**Pull request by yourbuddyconner**
- Adds a new Seed Node helm chart
- Refactored existing helm charts to support runtime genesis timestamp via `daemon.json`
- Refactored `values.yaml` to be more consistent across the charts

TODO: 
- Abstract the interface to `daemon.json` more, pass all runtime values via `daemon.json` as opposed to CLI Flags

### CODE REVIEW

File: helm/block-producer/templates/block-producer.yaml [hunk 1]
```diff
@@ -139,50 +139,34 @@ spec:
             value: {{ $.Values.faucetFee | quote }}
         imagePullPolicy: Always
       {{ end -}}
-      {{- if has (float64 $i) $.Values.blockProducersWithPoints }}
```
Comment (nholland94) [inline helm/block-producer/templates/block-producer.yaml:142]: Why was this deleted? We need this for the upcoming release.

File: helm/seed-node/templates/genesis-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+kind: ConfigMap
+metadata:
+  name: {{ tpl $.Values.seed.fullname . }}-genesis-config
+  labels:
+    app: {{ tpl $.Values.seed.fullname . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+  genesis_constants.json: |-
+    {
```
Comment (nholland94) [inline helm/seed-node/templates/genesis-configmap.yaml:12]: Can a subtemplate for the genesis constants data be shared between the block-producer and seed-node charts? If yes, I recommend doing the same for some of the common CLI flags. Or you could just leave it like this and wait for Dhall, where it should be arbitrary to setup these kinds of things.

File: terraform/modules/kubernetes/testnet/helm.tf [hunk 3]
```diff
@@ -17,58 +17,78 @@ provider helm {
 
 locals {
   seed_peers = [
-    "/ip4/${module.seed_one.instance_external_ip}/tcp/10001/ipfs/${split(",", module.seed_one.discovery_keypair)[2]}",
-    "/ip4/${module.seed_two.instance_external_ip}/tcp/10001/ipfs/${split(",", module.seed_two.discovery_keypair)[2]}"
+    "/dns4/seed-node.${var.testnet_name}/tcp/10001/ipfs/${split(",", var.seed_discovery_keypairs[0])[2]}"
```
Comment (nholland94) [inline terraform/modules/kubernetes/testnet/helm.tf:20]: Is there an easy way to make "kubernetes seeds" an option for now? We need the 	GCE instances for public testnets and mainnet anyway, right?

Comment (yourbuddyconner) [reply terraform/modules/kubernetes/testnet/helm.tf:20]: Yes, will implement

File: helm/seed-node/templates/genesis-configmap.yaml (hunk 2 above)
Comment (yourbuddyconner) [reply helm/seed-node/templates/genesis-configmap.yaml:12]: Yeah, I think the model I'd like to keep for now is separate helm charts in order to keep the template complexity down. Once Dhall lands this will all go away and it will be trivial to collapse all the coda-related charts into one.

File: helm/block-producer/templates/block-producer.yaml (hunk 1 above)
Comment (yourbuddyconner) [reply helm/block-producer/templates/block-producer.yaml:142]: Probably just an artifact of merging this PR with the upstream changes, didn't know what this was for explicitly. 

Brings up the discussion that the points should be its own helm chart and not be shoehorned into the block-producer deployment. I had been putting it off for the time being, but I can probably go ahead and add a new chart for points in this PR as well.

Comment (nholland94) [reply helm/block-producer/templates/block-producer.yaml:142]: Yeah, sorry about the ugly hacks. The user agent should probably become it's own chart as well. But I'm hesitant to change this before the next release at least, and think we should at minimum do it in a separate PR.

Comment (yourbuddyconner) [reply helm/block-producer/templates/block-producer.yaml:142]: Yeah cool, I will reinstate this diff in this PR and go ahead and refactor in a subsequent PR.
